webapp/src/main/java/com/box/l10n/mojito/service/tm/importer/TextUnitBatchImporterService.java

```diff
@@ -279,12 +304,17 @@ void applyIntegrityChecks(
             textUnitForBatchImport.setStatus(currentTextUnit.getStatus());
           } else {
             logger.info(
-                "Integrity check failed for string with source {}, content {}: {}",
+                "Integrity check failed for string with source:\n{}\n\nand content:\n{}",
```

Comment:
there is a warning in intellij saying "fewer arguments provided than placeholders specified." Wouldn't be better just use String.format instead, just to make sure?

Reply:
that's the right code though. If the last element is an exception, it will show the stack independently and not interpolating in the template. 

```
logger.info("template {}, {}, {}", 1, 2, exception)
```
will show

```
template 1, 2, {}
stack trace...
```
and not:
```
template 1, 2, message/stack trace...
```

https://www.slf4j.org/faq.html#paramException

Reply:
yes, my bad sorry, the warning was in the previous implementation. You actually fixed it now. Thank you